package.json

```diff
@@ -27,14 +27,14 @@
   },
   "homepage": "https://github.com/alpheios-project/data-models#readme",
   "devDependencies": {
-    "alpheios-node-build": "github:alpheios-project/node-build",
+    "alpheios-node-build": "github:alpheios-project/node-build#v0.6.3",
```

Comment:
May be we need to merge this version to master too?

Reply:
master probably should not have fixed dependency versions. Will fix.